x-pack/plugins/cases/server/client/cases/utils.ts

```diff
@@ -76,6 +77,13 @@ const getCommentContent = (comment: CommentResponse): string => {
   } else if (comment.type === CommentType.alert || comment.type === CommentType.generatedAlert) {
     const ids = getAlertIds(comment);
     return `Alert with ids ${ids.join(', ')} added to case`;
+  } else if (comment.type === CommentType.actions) {
```

Comment:
I think it will be a good idea to also add action.type in the condition too and same for switch below.

Reply:
@jonathan-buttner @cnasikas, we should start refactoring this code to be a factory to make it cleaner and easier to add new type.